src/Geometries/LinePrimitives/LinePrimitives.jl

```diff
@@ -17,10 +17,10 @@ struct Line{T, N, S} <: AbstractLine{T, N, S}
     dir::AbstractCoordinateVector{T, N, S}
 end
 
-function Line(p1::CartesianPoint{T}, p2::CartesianPoint{T})::Line{T, 3, :cartesian} where {T}
-    return Line{T, 3, :cartesian}(p1, p2 - p1)
+function Line(p1::CartesianPoint{T}, p2::CartesianPoint{T})::Line{T, 3, Cartesian} where {T}

```

Comment:
Are LinePrimitives used by anything? This is in the old geometry folder, are we keeping it? Surface interactions will require LinePrimitives down the line, but I rewrote these for the new CSG

Reply:
We should delete the old geometry folder at some point